### PR TITLE
common.xml: add VIBRATION msg

### PR DESCRIPTION
**Pull request by rmackay9**
Adds Vibration message as discussed with Tridge and Lorenz.

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -2776,6 +2776,16 @@
             <field type="float" name="distance">Distance to the target from the vehicle in meters</field>
         </message>
         <!-- MESSAGE IDs 180 - 240: Space for custom messages in individual projectname_messages.xml files -->
+        <message id="241" name="VIBRATION">
```
Comment (LorenzMeier) [inline message_definitions/v1.0/common.xml]: Can we keep this message in flux for 2-3 weeks until we know we have everything in it needs, or do you need to deploy like tomorrow?

**Comment by LorenzMeier** [comment]
Thanks!

**Comment by rmackay9** [comment]
I'd like to merge it into ardupilot master immediately.. but there won't be any consumers of it immediately.  I'd expect that will take a few weeks.. so I think we have some time to change it.  And we could do that change in the ardupilot area I think.

**Comment by LorenzMeier** [comment]
cool, merging.

**Comment by TSC21** [comment]
@LorenzMeier support has been add in mavros side - https://github.com/mavlink/mavros/pull/309. Please let us know of future usage of this message in both PX4 and Ardupilot so we can add more features to the plugin.

**Pull request merged**